ansys/dpf/core/plotter.py

```diff
@@ -74,6 +74,37 @@ def add_field(self, field, meshed_region=None, **kwargs):
         # plot
         self._plotter.add_mesh(meshed_region.grid, scalars=overall_data, **kwargs)
 
+        if show_max or show_min:
+            # Get Min-Max for the field
+            min_max = core.operators.min_max.min_max()
+            min_max.inputs.connect(field)
+
+        # Add Min and Max Labels
+        _labels = []
+        _grid_points = []
```

Comment:
Private variables within a class method aren't necessary. Because the namespace is limited to the method, we're not exposing anything and it's somewhat confusing to have a leading `_`, implying a private variable.

```suggestion
        labels = []
        grid_points = []
```